apps/vs-code-designer-react/src/app/DesignerCommandBar/index.tsx

```diff
@@ -42,6 +50,7 @@ export const DesignerCommandBar: React.FC<DesignerCommandBarProps> = ({ isRefres
       skipValidation: false,
       ignoreNonCriticalErrors: true,
     });
+    updateStandardApp(definition);
```

Comment:
you are only updating definition, wont you need to save parameters or connection references?

Reply:
ohh yeah, I will add those too. Thanks!

Reply:
I checked it and by saving the parameters and connection references just to the files doesn't cause any issue.